primitives/src/config/default_runtime_config.rs

```diff
@@ -11,7 +11,7 @@ use crate::{
 	config::WithExtrinsicParams, AssetRuntimeConfig, Config, GenericExtrinsicParams, PlainTip,
 };
 
-/// Default set of commonly used types by Polkadot nodes.
+/// Commonly used config of Substrate and Polkadot nodes.
 pub type DefaultRuntimeConfig =
```

Comment:
Similar to previous comment:  "Standard runtime config for Substrate and Polkadot nodes that use native payment" ?

Reply:
I'd refrain from using native payment, as Parity does not use it either. 

What about a simple `/// Standard runtime config for Substrate and Polkadot nodes.` ?